<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tbMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QSlider" name="hzColorV">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsBlurV">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsThreshold">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbColor">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbThreshold">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbBlur">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Blur</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbColor">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbBlur">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbThresh">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btOpenImage">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbImage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Name</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbLines">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbLines">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Lines</string>
      </property>
     </widget>
     <widget class="QSlider" name="hsThreshold_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hsThreshold_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbGrayscale">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Grayscale</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btSaveCommand">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Command</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btSaveSequence">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Sequence</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btRemoveCommand">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Command</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lsMain">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lnSequence">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>seq.conf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
